doc/CollisionSolverEntities.doc.md

```diff
@@ -0,0 +1,43 @@
+# Collision Solver for Entities other than Isaac
+
+## Label
+
+`collision`
+
+## Parameters
+
+| Label | Type | Size/Struct | Description |
+| ----- | ---- | ----------- | ----------- |
+| b | register | 1 byte | the 4 left bits are flags which indicates which side of the element the entity touched (right, left, lower, upper) |

```

Comment:
I don't think we know how to find this information yet, can you find a solution to make it works without?